CHANGELOG.md

```diff
@@ -40,6 +40,7 @@
 
 ### 🛡 Security
 - Update cross-spawn to address CVE ([#1469](https://github.com/opensearch-project/oui/pull/1469))
+- Update webpack and webpack-dev-server to address CVEs ([#1473](https://github.com/opensearch-project/oui/pull/1473))
```

Comment:
We'd want to to move the existing unreleased items to 1.18 and then add this to the unreleased section

Reply:
Makes sense, I'll update this